js/app.js

```diff
@@ -0,0 +1,203 @@
+// Answer the Following
+console.log("Answer the Following")
+console.log("\n");
+
+console.log("A light switch that can be either on or off.")
+console.log("Answer: boolean. let example = true;");
+console.log("\n");
+// A user's email address.
+console.log("A user's email address.")
+console.log("Answer: string. let example = 'sergio@sergio.com'");
+console.log("\n");
+// A spaceship with a hull, laser blasters, tractor beam, and warp drive.
+console.log("A spaceship with a hull, laser blasters, tractor beam, and warp drive.")
+console.log("Answer: object. let example = {hull: true, laserBlasters: true, tractorBeam: true, warpDrive: true}");
+console.log("\n");
+// A list of student names from our class.
+console.log("A list of student names from our class.")
+console.log("Answer: array. ['sergio','sergio','sergio','sergio']");
+console.log("\n");
+
+// A list of student names from our class, each with a location.
+console.log("A list of student names from our class, each with a location.")
+console.log("Answer: object. let example = {sergio: {location: Chicago}, sergio2: {location: Chicago}, sergio3: {location: Chicago}}");
+console.log("\n");
+
+// A list of student names from our class, each with a location and each with a list of favorite tv shows.
+console.log("A list of student names from our class, each with a location and each with a list of favorite tv shows.")
+console.log("Answer: object. let example = {sergio: {location: Chicago, favMovies: ['gsd','gsd','dgsdsf']}, sergio2: {location: Chicago, favMovies: ['gsd','gsd','dgsdsf']}, sergio3: {location: Chicago, favMovies: ['gsd','gsd','dgsdsf']}}");
+console.log("\n");
+console.log("\n");
+
```

Comment:
**6/6** good job
in general i love your attention to good output, but would have preferred you actually declare the things

Reply:
also what about the other students in the class LOL